ts/components/SessionWrapperModal.tsx

```diff
@@ -34,7 +34,11 @@ type WithExtraRightButton = {
 };
 type WithShowExitIcon = { showExitIcon?: boolean };
 
-const StyledModalHeader = styled(Flex)<{ bigHeader?: boolean; scrolled: boolean }>`
+const StyledModalHeader = styled(Flex)<{
+  bigHeader?: boolean;
+  scrolled: boolean;
+  flowContentIntoHeader?: boolean;
```

Comment:
is this used?

Reply:
its not, sorry I created this and reverted it, missed the argument tho, will remove